test/features/shopping.feature

```diff
@@ -1,4 +1,4 @@
-Feature: Shopping functionality
+Feature: Shopping functionality  
```

Comment:
koja je poenta ovog testa `Scenario: User is able to add and remove items from cart`?
Mozda je bolje da to razdvojis na dva testa...

Reply:
Pokrio sam sve nacine dodavanja (2 nacina) i brisanja (3 nacina) artikla:
1. Dodam iz homePage-a, obrisem iz istog. 
2. Dodam iz itemPage-a, obrisem iz istog.
3. dodam iz itemPage-a, obrisem iz checkoutPage-a.
Svakako cu poslusati tvoju sugestiju, al samo sam hteo da ti objasnim poentu testa